profilegen/src/profile_templates.rb

```diff
@@ -0,0 +1,34 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative 'profile_template_files'
```

Comment:
why not put the file variables here instead? Anyways we are creating static methods of the class `ProfileTemplates` ?

Reply:
Personal choice. Not changing